pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -14,6 +14,12 @@ python3Packages.buildPythonApplication rec {
     fetchSubmodules = true;
   };
 
+  patches = [(fetchpatch {
```

Comment:
please add a comment as to when this can be removed.
```suggestion
  patches = [
    # merged to master, remove next version bump
    (fetchpatch {
```

Reply:
OK, done